extension-manifest-v3/src/utils/engines.js

```diff
@@ -40,6 +40,10 @@ const ENV = new Map([
 
 observe('experimentalFilters', (experimentalFilters) => {
   ENV.set('env_experimental', experimentalFilters);
+
+  // As engines on the server might have new filters, we need to update them
+  updateAll().catch(() => null);
```

Comment:
should't this be awaited?

Reply:
Don't think so. The effect will be the same.